ui/app/helpers/date-from-now.js

```diff
@@ -1,8 +1,17 @@
 import { helper } from '@ember/component/helper';
-import { distanceInWordsToNow } from 'date-fns';
+import { formatDistanceToNow } from 'date-fns';
 
 export function dateFromNow([date], options = {}) {
-  return distanceInWordsToNow(date, options);
+  // debugger;
+  // const time = fromUnixTime(date / 1000);
+  let dateString;
+  try {
+    dateString = formatDistanceToNow(date, { ...options });
+  } catch (e) {
+    console.log(date);
```

Comment:
maybe also console log the error, or is this just for testing?

Reply:
this is just for testing -- we haven't solved the issue yet 😭